tap_anthem/streams.py

```diff
@@ -1,62 +1,149 @@
 """Stream type classes for tap-anthem."""
 
-from pathlib import Path
-from typing import Any, Dict, Optional, Union, List, Iterable
+from typing import Iterable
 
+import requests
 from singer_sdk import typing as th  # JSON Schema typing helpers
 
-from tap_anthem.client import anthemStream
+from tap_anthem.client import AnthemStream, FormularyNavigatorStream
 
-# TODO: Delete this is if not using json files for schema definition
-SCHEMAS_DIR = Path(__file__).parent / Path("./schemas")
-# TODO: - Override `UsersStream` and `GroupsStream` with your own stream definition.
-#       - Copy-paste as many times as needed to create multiple stream types.
 
-
-class UsersStream(anthemStream):
+class NhProvidersStream(AnthemStream):
     """Define custom stream."""
-    name = "users"
-    path = "/users"
-    primary_keys = ["id"]
-    replication_key = None
-    # Optionally, you may also use `schema_filepath` in place of `schema`:
-    # schema_filepath = SCHEMAS_DIR / "users.json"
+    name = "nh_providers"
+    path = '/PROVIDERS_NH.json'
+    primary_keys = ["npi"]
+    replication_key = "last_updated_on"
```

Comment:
Refer to [the Meltano docs](https://docs.meltano.com/guide/integration#replication-key) on why the `replication_key` matters.